src/redux/reducers/authReducer.ts

```diff
@@ -1,7 +1,7 @@
 "use client";
 import { createSlice } from "@reduxjs/toolkit";
 import type { PayloadAction } from "@reduxjs/toolkit";
-import { setCookie } from "nookies";
+import { setCookie } from "cookies";
```

Comment:
you cannot directly access the cookies in nextjs client. That's why added the nookies package.